src/schema/organization/contributors/resolvers.js

```diff
@@ -5,6 +5,8 @@ import { getOldestDate } from '../../../utils';
 import 'dotenv/config';
 import config from '../../../config';
 
+const nonValidUsers = ['gitter-badger']
```

Comment:
```suggestion
const nonValidUsers = ['gitter-badger'];
```
Adicionar um `;` no final, pra ficar de acordo com o padrão que a gente adotou